python-threatexchange/tests/hashing/test_tlsh_hasher.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
This folder is to test the hash lib outside of signal_type:
can you add it to the same folder as: 
`python-threatexchange/threatexchange/signal_type/tests/test_md5_hash.py`
and follow its pattern (test should run just the same, it'll just be closer to the file it is testing)